tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Hooks/IntegrationHook.cs

```diff
@@ -25,6 +29,12 @@ public async Task BeforeScenarioAsync()
             RegisterTestConfiguration();
             RegisterCustomValueRetrievers();
 
+            if (_firstScenario)
```

Comment:
Shouldn't need this boolean if we use the attribute [BeforeTestRun]. Put this on top of a new method & it will be called before all tests run.

Reply:
[BeforeTestRun] needs the method to be static, so if we used this then we would have to hardcode the baseUrl and the time expiry